doc/languages-frameworks/dotnet.section.md

```diff
@@ -73,6 +73,8 @@ To package Dotnet applications, you can use `buildDotnetModule`. This has simila
 
 * `projectFile` has to be used for specifying the dotnet project file relative to the source root. These usually have `.sln` or `.csproj` file extensions. This can be an array of multiple projects as well.
 * `nugetDeps` has to be used to specify the NuGet dependency file. Unfortunately, these cannot be deterministically fetched without a lockfile. This file should be generated using `nuget-to-nix` tool, which is available in nixpkgs.
+* `packNupkg` is used to pack project as a nupkg and save it in `$out/share`. If set to `true`, this derivation can be used as a dependency in `projectReferences` for another dotnet project.
+* `projectReferences` has to be used to resolve `ProjectReference` project items. Referenced projects can be packed with `buildDotnetModule` builder with `packNupkg=true` attribute and passed in a list in `projectReferences`. NuGet packages that are passed to `projectReferences` attribute are discovered by `nuget init` command.
```

Comment:
For attributes that aren't strictly required for all derivations, we shouldn't word it as `has to be used`. 

I also think we shouldn't need to communicate how we instantiate these dependencies, since the process is entirely abstracted away. I might be missing a case where this information is useful though.

```suggestion
* `projectReferences` can be used to resolve `ProjectReference` project items. Referenced projects can be packed with `buildDotnetModule` by setting the `packNupkg = true` attribute and passing a list of derivations to `projectReferences`.
```

Reply:
fixed in the updated commit